conn_linux.go

```diff
@@ -226,6 +226,16 @@ func (c *conn) SetBPF(filter []bpf.RawInstruction) error {
 	)
 }
 
+// DetachBPF removes a BPF filter from a conn.
+func (c *conn) DetachBPF() error {
+	var dummy int
```

Comment:
This should be `uint32`, and let's leave a comment explaining that the option value is ignored.